pkg/services/live/pipeline/data_output_builtin.go

```diff
@@ -0,0 +1,47 @@
+package pipeline
+
+import (
+	"context"
+	"errors"
+
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/live/livecontext"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+type BuiltinDataOutput struct {
+	channelHandlerGetter ChannelHandlerGetter
+}
+
+const DataOutputTypeBuiltin = "builtin"
```

Comment:
instead of builtin we'd want output like `database` (to save message in db for broadcast scope). But this is an extra work to decouple so for now its generic `builtin` with a help of `ChannelHandlerGetter`.

Reply:
I assume "builtin" would apply to the whole rule/channel?